lib/transport/ws.ex

```diff
@@ -42,7 +42,13 @@ defmodule Janus.Transport.WS do
 
     opts =
       opts
-      |> Keyword.update(:extra_headers, [janus_protocol], &[janus_protocol | &1])
+      |> Keyword.update(:extra_headers, [janus_protocol], fn headers ->
+        if List.keymember?(headers, "Sec-WebSocket-Protocol", 0) do
+          headers
+        else
+          [janus_protocol | headers]
+        end
+      end)
```

Comment:
I'm not a big fan of this solution - it should not be the responsibility of library user to know the protocol Janus is using for Admin API. I don't know how you're going to provide handlers for Admin messages, but there should be either an option passed to connection that would change `janus-protocol` -> `janus-admin-protocol` or the header should be obtained from a module (if you're passing different modules for both scenarios). There may be other solutions, I'm open for suggestions, however manually passing this header in some cases is not a good one.